tasks/post_to_catalog/test/unit_tests/test_post_to_catalog.py

```diff
@@ -83,6 +83,7 @@ def test_send_record_to_database_happy_path(
             "version": uuid.uuid4().__str__(),
         }
         granule = {
+            "providerId": uuid.uuid4().__str__(),
```

Comment:
Per my comment on the schema, these conflict with provider["providerId"]. Tests shouldn't need any additional input.

Reply:
changed to `"provider_id": provider["providerId"]`